src/main/proto/marathon.proto

```diff
@@ -148,11 +149,42 @@ message ExtendedContainerInfo {
   }
 
   required mesos.ContainerInfo.Type type = 1;
-  repeated mesos.Volume volumes = 2;
+  repeated Volume volumes = 2;
```

Comment:
I have tested with this app definition
```
{
  "id": "docker-with-vol",
  "instances": 1,
  "container": {
    "type": "DOCKER",
    "docker": {
      "image": "thomasr/sleep"
    },
    "volumes": [{
      "containerPath": "/my-mount",
      "hostPath": "/tmp",
      "mode": "RW"
    }]
  }
}
```

Created in 0.15.1, still working in 0.16.0-SNAPSHOT and can be scaled/restarted.